tee-worker/cli/src/trusted_operation.rs

```diff
@@ -461,6 +384,7 @@ pub(crate) fn send_direct_batch_vc_request(
 
 	debug!("waiting for rpc response");
 	let mut req_cnt = 0u8;
+	let mut ignore_first = true;
 	std::thread::spawn(move || loop {
```

Comment:
> Can we do this inside send_direct_batch_vc which should collect responses, and give a vector of responses here and we can just print them. wdyt?

If we want to do this as mentioned, then we should probably not move to different thread.